src/selector.py

```diff
@@ -70,15 +79,18 @@ class LabelSelector(Selector):
     A class representing a single label selector as described here:
     https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#set-based-requirement
     """
-    def __init__(self, key, op, values):
+
+    def __init__(self, key, op, values, calico_selector_expr):
         """
         :param str key: Label key
         :param SelectorOp op: The selector operator
         :param list values: A list of label values
+        :param str calico_selector_expr: the Selector expr by Calico syntax
         """
         self.key = key
         self.operator = op
         self.values = values
+        self.calico_selector_expr = calico_selector_expr
```

Comment:
Better produce the calico-style selector on-the-fly

Reply:
done